tracer/build/crank/Samples.AspNetCoreSimpleController.yml

```diff
@@ -69,29 +69,6 @@ scenarios:
         serverPort: 5000
         path: /hello
 
-  manual_only:
```

Comment:
Hmm I don't know where would be appropriate, but I think it could still be good to have some micro-benchmark or macro-benchmark that shows that there is no overhead when only manual instrumentation is added to an application

Reply:
Hmmm, yeah, fair point 🤔 In which case, maybe we should reinstate this, but only run it in the "extended" case, because we don't ever expect it to change?

Reply:
Reinstated it here: https://github.com/DataDog/dd-trace-dotnet/pull/5183/commits/101ff8fd7d5d8105f4a811b32236c58dbed1a363